CONTRIBUTING.md

```diff
@@ -1,7 +1,19 @@
 # How to Contribute
 
-Hello World, all your contributions are welcome! 
```

Comment:
**Hello World .... just fork first.**
We should not delete that, beginners need to understand how it works.